src/thinblock.cpp

```diff
@@ -76,24 +78,24 @@ bool CThinBlock::HandleMessage(CDataStream &vRecv, CNode *pfrom)
         if (mi == mapBlockIndex.end())
         {
             Misbehaving(pfrom->GetId(), 10);
-            return error("thinblock from peer %s (%d) will not connect, unknown previous block %s",
-                pfrom->addrName.c_str(), pfrom->id, prevHash.ToString());
+            return error("thinblock from peer %s (%d) will not connect, unknown previous block %s", pfrom->GetLogName(),
+                pfrom->id, prevHash.ToString());
```

Comment:
GetLogName will return "IP (id)" if names are allowed, but just "id" if they are not, so it is redundant to have the "(%d)" parameter.  The printf construct should look like:
"thinblock from peer %s will not connect, unknown previous block %s", pfrom->GetLogName(), prevHash.ToString());

this issue exists in a few places in this PR, please search for them...